docs/available-filters/multi-tenancy/multi-tenancy.adoc

```diff
@@ -1 +1,63 @@
+:github: https://github.com/kroxylicious/kroxylicious
 = Multi-tenancy
+
+== What is it?
+
+Multi-tenancy filter enables isolation prefixing each metadata transiting through virtual cluster to target cluster.
+See https://kafka.apache.org/documentation/#multitenancy[Apache Kafka multi-tenancy documentation] for more information.
+
+== How to use the filter
+
+There are two steps to using the filter.
+
+1. <<Configuring virtual clusters>>
+2. Configuring the filter within Kroxylicious.
+
+=== Configuring the filter within Kroxylicious.
+
+[source, yaml]
+----
+filters:
+  - type: MultiTenantTransformationFilterFactory
+    config:
+      prefixResourceNameSeparator: "." #<1>
+----
+<1> `-` is the default separator if no config is provided
+
+If the virtual cluster name is `demo`, the created prefix will be `demo.`
+
+NOTE: Currently, only the prefix with separator is validated.
+
+=== Verifying that multi-tenancy is occurring
+
+Set up two virtual clusters `devenv1` & `devenv2`. See this {github}/blob/main/kubernetes-examples/multi-tenant/base/kroxylicious/kroxylicious-config.yaml[example].
+
+Create a topic on `devenv1`
+
+[source]
+----
+kafka-topics.sh --bootstrap-server devenv1:9392 --create --if-not-exists --topic first-topic
+----
+
+Create a topic on `devenv2`
+
+[source]
+----
+kafka-topics.sh --bootstrap-server devenv2:9392 --create --if-not-exists --topic second-topic
+----
```

Comment:
Would it make for a clearer demo if we used the same topic name in each virtual cluster? for example 
```suggestion
----
kafka-topics.sh --bootstrap-server devenv1:9392 --create --if-not-exists --topic test-topic
----

Create a topic on `devenv2`

[source]
----
kafka-topics.sh --bootstrap-server devenv2:9392 --create --if-not-exists --topic test-topic
----
```

Reply:
In fact all options are valid, using same topic name demonstrates that the creations works on the two steps. Using different names is to demonstrate when you list topics you don't see the one from the other virtual cluster

Reply:
Yeah, I think your right for the purposes of this it makes sense to use different names so that listings are clearer.